### DISCUSSION TITLE
Disable SuluAudienceTargetingBundle

### DISCUSSION THREAD

**Discussion by juagarc4** [sulu/sulu#6923] [Q&A] [answered]
# Problem description
Our technical department has complained about the many requests to /_sulu_target_group_hit made by the SuluAudienceTargetingBundle. 
We don't use this bundle and want to deactivate it, but all attempts result in error.
How can we disable this module without causing errors on the page?

Thanks in advance
Regards

**Comment by alexander-schranz** [comment, marked as answer]
@juagarc4 The SuluAudienceTargetingBundle is not enabled by default. If you did enable it you can remove it from your `config/bundles.php`.

You need also remove all kind of the installation configs listed here: https://docs.sulu.io/en/2.5/bundles/audience_targeting.html:

 - config/bundles.php
 - config/routes/sulu_admin.yaml
 - config/routes/sulu_website.yaml
 - src/Kernel.php
